README.md

```diff
@@ -1,16 +1,55 @@
 ## QA/Samples Repository
 
-This application is intended to be used for testing purposes. It contains samples related to the various [EnyoJS](http://enyojs.com) [libraries](https://github.com/enyojs). Please follow the directions below to build and run the tests.
+This application is intended to be used for testing purposes. It contains samples related to the
+various [EnyoJS](http://enyojs.com) [libraries](https://github.com/enyojs). Please follow the
+directions below to build and run the tests.
 
-This application expects that you have the [enyo-dev](https://github.com/enyojs/enyo-dev) module installed.
+### Build
 
-> At this time, it is not released as an npm package thus you must clone it locally on the system, execute `npm install -g` for the eqivalent.
+#### enyo-dev Tooling
 
-Ensure all of the dependent libraries are installed by executing `egen init`. The `egen` executable is an alias to `enyo-gen` from the [enyo-dev](https://github.com/enyojs/enyo-dev) module. By default, this will install all of the required dependencies from their respective git repositories (will be slow on most connections, so be patient). If you already have a local copy of all of the libraries it will be faster with the `egen init --link-all-libs` flag.
+This application requires the [enyo-dev](https://github.com/enyojs/enyo-dev) tools installed in
+order to build and run. At this time, it is not released as an npm package thus you must clone it
+locally on the system, execute `npm install -g` for the eqivalent. Once installed, the `enyo`
+command will be available to initialize and build Enyo projects.
 
-**Note: `--link-all-libs` can only be used if you have also executed `bower link` from within each of those repositories. You can also only link specific libraries if you do not have them all: `egen init --link-libs=enyo,moonstone`.**
+```
+npm clone https://github.com/enyojs/enyo-dev
+cd enyo-dev
```

Comment:
And that's why you get to do our doc work; you're so much better than the rest of us! :stuck_out_tongue_winking_eye: 

I don't think adding an npm command that globally installs a *different module globally* is appropriate. Instead, I've updated the above with links to the enyo-dev docs rather than duplicate instructions. wdyt?